packages/dev/core/src/Animations/animationGroup.ts

```diff
@@ -59,6 +60,16 @@ export class AnimationGroup implements IDisposable {
     private _speedRatio = 1;
     private _loopAnimation = false;
     private _isAdditive = false;
+    private _playOrder = 0;
+
+    /** Order of play so that the animatables of this animation group are played before all other animatables */
+    public static readonly PLAYORDER_FIRST = Constants.ANIMATABLE_PLAYORDER_FIRST;
+
+    /** Order of play so that the animatables of this animation group are played after all other animatables */
+    public static readonly PLAYORDER_LAST = Constants.ANIMATABLE_PLAYORDER_LAST;
+
+    /** Enables automatic sorting of animation groups according to play order (default: false) */
+    public static EnableAutomaticPlayOrderSorting = false;

```

Comment:
should it be static or should we assign a default number in creation order (if possible) ???

Reply:
I'm not sure I understand, but since animation groups can be played/stopped at any time, the animatables in the `scene._activeAnimatables` array won't be in the right order, so we can't do without sorting.

Reply:
After our discussion, I removed the flag and sort if necessary, as sorting is unlikely to be a performance issue, given the numbers involved and the fact that we only sort when an animation group is started and not at every frame.